src/headers/tomcrypt_math.h

```diff
@@ -339,10 +355,15 @@ typedef struct {
        @param G   The point to multiply
        @param R   The destination for kG
        @param modulus  The modulus for the field
-       @param map Boolean indicated whether to map back to affine or not (can be ignored if you work in affine only)
+       @param map Boolean indicated whether to map back to affine or not
+                  (can be ignored if you work in affine only)
        @return CRYPT_OK on success
    */
-   int (*ecc_ptmul)(void *k, ecc_point *G, ecc_point *R, void *modulus, int map);
+   int (*ecc_ptmul)(     void *k,
```

Comment:
please avoid changing anything related to `ecc_*` especially when it is just a cosmetic change like this. It will cause a merge errors once I'll be trying to rebase `miko-ecc-enhancements` (I had to modify most of `ecc_*` functions in `tomcrypt_math.h`)